package.json

```diff
@@ -31,7 +31,7 @@
     "@types/deep-freeze": "^0.1.1",
     "@types/express": "4.16.0",
     "@types/express-session": "^1.15.12",
-    "@types/lodash": "^4.14.122",
+    "@types/lodash": "4.14.122",
```

Comment:
Is that deliberate? Why did you need to pin the version?

Reply:
It's necessary because it changed `isObject` to be a true type guard so that
```ts
export const resolveOdataBind = (odataBinds: ODataBinds, value: any) => {
	if (_.isObject(value) && value.bind != null) {
		[, value] = odataBinds[value.bind];
	}
	return value;
};
```
now fails because `Property 'bind' does not exist on type 'object'.`